source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -361,6 +374,23 @@ The color of the product
 ~~~
 
 
+<% if settings.feed_validator_domain %>
+## Data Feed Validator
+
+You can use our public <%= link_to "Data Feed Validator",
+"http://#{settings.feed_validator_domain}" %> to check your XML feed for
+compatibility with <%= settings.site_name.capitalize %> standards.
+
+There, you can upload your XML file and the validator checks to confirm that it is
+well-formed according to the XML specification, sees if it can identify the XML
+element corresponding to each one of your products and then scans the products'
+attributes to ensure that <%= settings.site_name.capitalize %> requirements are met.
+
+<%= link_to image_tag("#{settings.site_name}/validator.png", class: 'img-responsive'),
+      "http://#{settings.feed_validator_domain}"  %>
+<% end %>
```

Comment:
The image you have added is more like a banner than a call to action. A call to action would be something like "Validate your XML" (i.e. a button). If you make the change locally you'll see it makes sense. Give it a shot. If we need a call to action, we should use (real) a BS button instead.